cql-unit-tests/test/treatmentSummary.test.ts

```diff
@@ -11,11 +11,9 @@ import { buildAllNullRiskAssessmentScoreParameters } from '../helpers/builders';
 describe('treatment summary', () => {
     it.each([
         ['Recommend Non-Pharmacologic Treatment'],
-        ['Recommend Antibodies Treatment'],
-        ['Recommend Anticoagulation Treatment'],
-        ['Recommend Steroids Treatment'],
-        ['Recommend Remdesivir Treatment'],
-        ['Recommend SteroidsAndOrRemdesivir Treatment'],
+        ['Recommend Admission Treatment'],
+        ['Recommend Discharged Treatment'],
```

Comment:
Discussed with @rocio-desantiago in the zoom to rename this to `Recommend Discharge Treatment` for clarity.